river/Seat.zig

```diff
@@ -55,6 +56,9 @@ cursor: Cursor = undefined,
 /// Mulitple keyboards are handled separately
 keyboards: std.TailQueue(Keyboard) = .{},
 
+/// There can be two switches: lid switch and tablet mode switch
+switches: std.TailQueue(Switch) = .{},
```

Comment:
Does this comment mean that this list of switches can have at most two items? If so it might better to just have `lid_switch` and `tablet_switch` fields. If not I think it should be reworded to avoid confusion.

Reply:
There are probably no devices that have more than one of each, but one could have additional switches in terms of uinput devices. Would we need an extra Switch in this case?